scripts/e2e/spm.sh

```diff
@@ -7,7 +7,7 @@ set -euf -o pipefail
 
 print_help() {
   echo "Usage: $0 [-b binary]"
-  echo "-b: Which binary to build: 'all-in-one' or 'jaeger' (v2) (default)"
+  echo "-b: Which binary to build: 'all-in-one', 'jaeger' (v2, default), or 'elasticsearch'"
```

Comment:
`elasticsearch` is not a binary. This should be a separate dimension like "database" (`-d elasticsearch`)